frontend/src/components/pages/LogRecords.tsx

```diff
@@ -1,106 +1,281 @@
-import React, { useState } from "react";
```

Comment:
At this point, this file can probably be renamed to `Homepage.tsx`. Also, I'd create a new dir in /pages called "Homepage" and move this file and the `LogRecordsTable.tsx` file into the new dir. 

For me, the common dir is for reusable components, and it makes more sense to group components that are used once together in their own dir. 

That's my thought process, I don't want devs that are unfamiliar with this code to go "huh??? why are there two log record files". What are your thoughts on this?

Reply:
Resolved ; we now have a `/HomePage` component within the `/pages` directory, and within that we have:
- HomePage.tsx (main entrypoint)
- LogRecordsTable.tsx (table component)
- SearchAndFilters.tsx (filtering component)